Libraries/Text/TextInput/Singleline/RCTUITextField.m

```diff
@@ -18,7 +18,7 @@ @interface RCTUITextFieldCell : NSTextFieldCell
 @property (nonatomic, assign) UIEdgeInsets textContainerInset;
 @property (nonatomic, getter=isAutomaticTextReplacementEnabled) BOOL automaticTextReplacementEnabled;
 @property (nonatomic, getter=isAutomaticSpellingCorrectionEnabled) BOOL automaticSpellingCorrectionEnabled;
-@property (nonatomic, strong, nullable) UIColor *selectionColor;
+@property (nonatomic, strong, nullable) RCTUIColor *selectionColor;
```

Comment:
Missing the TODO comment

Reply:
This change is already inside a ` // [TODO(macOS ISS#2323203)` ... ` // ]TODO(macOS ISS#2323203)` block.